CMakeLists.txt

```diff
@@ -90,7 +95,6 @@ endif()
 configure_file("${CMAKE_SOURCE_DIR}/src/version.h.in" "${CMAKE_BINARY_DIR}/generated/version.h" @ONLY)
 include_directories(${CMAKE_BINARY_DIR}/generated)
 
-set(CMAKE_SOURCE_DIR "${CMAKE_BINARY_DIR}/../")
```

Comment:
I'm not sure why this line was added in the first place, but it is problematic because Visual Studio places the binary directory several levels deeper.